whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -1391,8 +1391,111 @@
 
     "relation_from_w": {
       "computeLinks": {"use": "$w", "mapping": {"g": "broader", "h": "narrower", "*": "related"}}
-    }
+    },
+
+    "referencedObjectDetails": {
+      "pendingResources": {
+        "_:contribution": {
+          "about": "_:work",
+          "addLink": "contribution",
+          "resourceType": "PrimaryContribution",
+          "embedded": true
+        },
+        "_:agent": {
+          "about": "_:contribution",
+          "link": "agent",
+          "resourceType": "Agent"
+        },
+        "_:workTitle": {
+          "about": "_:work",
+          "addLink": "hasTitle",
+          "resourceType": "Title"
+        },
+        "_:title": {
+          "about": "_:instance",
+          "addLink": "hasTitle", "resourceType": "Title",
+          "embedded": true
+        },
+        "_:describedBy": {
+          "NOTE": "If not possible to link, LinkFinder will change it to a Local identifiedBy property",
+          "about": "_:instance",
+          "addLink": "describedBy", "infer": true,
+          "resourceType": "Record"
+        }
+      },
+      "i1": {
+        "NOTE": "By moving 580 to 500 this indicator will not be usefull anymore",
+        "marcDefault": "0"
+      },
+      "$a": {"about": "_:agent", "property": "label", "infer": true, "TODO:targetMatch": "1xx"},
+      "$b": {"about": "_:instance", "property": "editionStatement", "TODO:targetMatch": "250"},
+      "$d": {"about": "_:instance", "property": "provisionActivityStatement", "TODO:targetMatch": "260$c"},
+      "$g": {"about": "_:instance", "addProperty": "part"},
+      "$h": {"about": "_:instance", "addLink": "extent", "resourceType": "Extent", "property": "label", "NOTE:marc-repeatable": false},
+      "$i": {"about": "_:work", "addProperty": "marc:displayText"},
```

Comment:
AFAICS, since the extracted pattern is consequently wrong (using the instanceOf->LINKREF->hasInstance we need to get rid of), and scheduled to be fixed in the next round, marc:displayText as used in this pattern reflects that.

Unfortunately, this breaks the usage of marc:displayText in otherPhysicalFormat (in e.g. the digitization workflows and associated scripts), which has up until this PR been correctly located next to describedBy. And we cannot change that. (It would break practise *and* the scripts.)

So we cannot land this PR until marc:displayText is put on `_:instance` and, presumably, the change script adjusts that everywhere, ensuring that is left in the correct place for otherPhysicalFormat.

Reply:
The simplest approach is probably to declare a sub-pattern for the general "displayText on works" and use that on all fields except 776, which keeps marc:displayText. Thus we don't need to rewrite much this round.

Reply:
The simplest approach is probably to move the "marc:displayText on works" to the `referenceToWork` sub-pattern, and define an "marc:displayText on instance" on referenceToInstance. Thus we don't need to rewrite much this round.

Reply:
I agree. The reason why I moved otherPhysicalFormat.marc:displayText to the work entity is to handle all the marc:displayText in the same manner. And after that, complete the plan to gradually resolve marc:displayText completely. But I agree, it's an unnecessary work to move otherPhysicalFormat, especially when marc:displayText should be together with describedBy and breaks existing scripts (which I was not aware of).

Reply:
Great! I addressed this in ff275d5edabbf047b3aab06597e94c4859bc4502.